plugins/modules/cloud/misc/proxmox_template.py

```diff
@@ -22,45 +22,45 @@
     type: str
   src:
     description:
-      - path to uploaded file
-      - required only for C(state=present)
+      - Path to uploaded file.
+      - Required only for I(state=present).
```

Comment:
FYI: this change (and some more of them) also appears in #5237.

Reply:
exactly because I branched off a branch that was not the main one - I see it adds quite a bit of confusion to it, but if we merge 5237 first I reckon this one would clear up quite a bit

Reply:
The main danger is that there will be conflicts. But apparently there aren't, so ... :)